docs-web/app-protect/troubleshooting.md

```diff
@@ -102,4 +106,21 @@ This timeout should be more than enough to verify configurations. However, when
 - You need to apply a large amount of Ingress Resources at once.
 - You are running the Ingress Controller for the first time in a cluster where the Ingress Resources with App Protect enabled are already present.
 
-You can increase this timeout by setting the `nginx-reload-timeout` [cli-argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-nginx-reload-timeout).
\ No newline at end of file
+You can increase this timeout by setting the `nginx-reload-timeout` [cli-argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-nginx-reload-timeout).
+
+If You are using external references in Your Nginx App Protect policies, verify if the servers hosting the referenced resources are available and that their response time is as short as possible (see next paragraph).
+If the references are not available during ingress controller startup the pod will fail to startup. In case the resources are not available during a reload, the relod will fail, and ingress controller will process traffic with the previous correct coonfiguration.
```

Comment:
```suggestion
If the references are not available during ingress controller startup the pod will fail to startup. In case the resources are not available during a reload, the reload will fail, and ingress controller will process traffic with the previous correct configuration.
```